Libraries/oneDAL/IntelPython_daal4py_Getting_Started/License.txt

```diff
@@ -0,0 +1,8 @@
+Copyright Intel Corporation
+
```

Comment:
Please change the text of the license file to the one [here](https://github.com/oneapi-src/oneAPI-samples/blob/master/License.txt)